andela/kissa-rc
# #137077471 Integrate Coveralls

#### What does this PR do?
iit integrates coveralls into app
#### Description of Task completed?
Added relevant modules and scripts to get coverage report and share online
#### How should this be manually tested?
Visit the repository's readme and find Badge, Click on badge to see details of coverage report
#### Any background context you want to provide?
#### What are the relevant pivotal tracker stories?
[#137077471 Integrate Coveralls code coverage service](https://www.pivotaltracker.com/story/show/137077471)
#### Screenshots (if appropriate)
#### Questions:
